db/migrate/20220331093754_fix_development_encoding_issues.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+# The default database encoding for the development database was wrong
+# We've fixed it up for newly created databases, but rather than forcing
+# developers to rebuild their database we'll set up a special development
+# only migration to keep things clean. This will:
+# - Update the default encoding
+# - Fix the incorrect encoding on any tables
+# Production and UAT are both correct, so we don't try and do anything there to
+# be safe.
+class FixDevelopmentEncodingIssues < ActiveRecord::Migration[6.0]
+  include MigrationExtensions::EncodingChanges
+
+  def up
+    say 'Checking Environment'
+    if migrate?
+      say "#{Rails.env} found, proceeding"
+      update_encoding
+      update_tables
+    else
+      say "#{Rails.env} found, skipping"
```

Comment:
The migration will only run in development/test. This is as production et al are setup correctly and we want to avoid accidentally introducing changes.